lib/full-schema.js

```diff
@@ -98,17 +98,21 @@ export const marks = {
 
   highlight: {
     attrs: {
-      'highlight-keys': {},
+      'highlight-key': {},
```

Comment:
You are modifying schema. This means that old documents stored in the database are not valid anymore!

We have to make a migration for this.